### PR TITLE
Add clonable to experimental features

### PR DESCRIPTION
**Pull request by pepelsbey**
### Description

- Adds `clonable` to experimental sections
- Cleans up empty titles

### Motivation

To support Fx 122 release.

### Related issues and pull requests

- https://github.com/mdn/content/issues/31108

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/122/index.md [hunk 1]
```diff
@@ -29,48 +25,26 @@ This article provides information about the changes in Firefox 122 that affect d
...
 #### Removals
 
 - Removed support for `data:` URLs in SVG `<use>` elements and via the [`SVGUseElement`](/en-US/docs/Web/API/SVGUseElement) interface to prevent [XSS](/en-US/docs/Glossary/Cross-site_scripting) attacks.
   The legacy functionality may be re-enabled by setting the `svg.use-element.data-url-href.allowed` preference to `true`, although this is not recommended for security reasons ([Firefox bug 1806964](https://bugzil.la/1806964)).
 
-### HTTP
-
-#### Removals
-
-### Security
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/122/index.md:51]: We've left `### Security` in before [with "No notable changes"](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/118#security) but I think this is inconsistent, so not blocking.

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 2]
```diff
@@ -1483,6 +1483,22 @@ It is disabled by default on all builds [Firefox bug 1750902](https://bugzil.la/
...
+- The {{domxref("ShadowRoot")}} interface's {{domxref("ShadowRoot.clonable", "clonable")}} read-only property returns `true` if the shadow root is clonable, and `false` otherwise.
+
+When shadow root is created via declarative shadow DOM, the `clonable` option is set to `true` by default, and the `clonable` property returns `true`. ([Firefox bug 1868428](https://bugzil.la/1868428))
+
+| Release channel   | Version added | Enabled by default? |
+| ----------------- | ------------- | ------------------- |
+| Nightly           | 122           | Yes                 |
+| Developer Edition | NA            | No                  |
+| Beta              | NA            | No                  |
+| Release           | NA            | No                  |
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/experimental_features/index.md:1500]: I presume there's no preference to set, right?

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 3]
```diff
@@ -1483,6 +1483,22 @@ It is disabled by default on all builds [Firefox bug 1750902](https://bugzil.la/
 
 ### HTML DOM API
 
+#### Shadow DOM
+
+Firefox now supports the `clonable` option and property for shadow DOM.
+
+- The {{domxref("Element.attachShadow()")}} method's `clonable` boolean option specifies whether the created shadow root is clonable: when set to `true`, the shadow host cloned with {{domxref("Node.cloneNode()")}} or {{domxref("Document.importNode()")}} will include shadow root in the copy.
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/experimental_features/index.md:1490]: Want to add "a"/"the"?

```suggestion
- The {{domxref("Element.attachShadow()")}} method's `clonable` boolean option specifies whether the created shadow root is clonable: when set to `true`, the shadow host cloned with {{domxref("Node.cloneNode()")}} or {{domxref("Document.importNode()")}} will include the shadow root in the copy.
```

**Review by bsmth — APPROVED** [review]
Looks good to me, thanks. Some minor non-blocking comments for you 👍🏻

File: files/en-us/mozilla/firefox/experimental_features/index.md (hunk 2 above)
Comment (pepelsbey) [reply files/en-us/mozilla/firefox/experimental_features/index.md:1500]: Yep, no preference this time.

File: files/en-us/mozilla/firefox/releases/122/index.md (hunk 1 above)
Comment (pepelsbey) [reply files/en-us/mozilla/firefox/releases/122/index.md:51]: I meant to bring this up. The “No notable changes” note might not be very useful. I’ll dare to skip it this time 😅

**Pull request merged**